proto/data_objects.proto

```diff
@@ -0,0 +1,268 @@
+
+syntax = "proto3";
+
+package ga4gh_data_object;
+
+// Import HTTP RESTful annotations
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+///-------------------------------------------------------------------
+message DataBundleObject {
+  //REQUIRED
+  string id = 1;
+
+  //REQUIRED
+  repeated DataObject files = 2;
+
+  //REQUIRED
+  // Timestamp of object creation in RFC3339.
+  google.protobuf.Timestamp created = 3;
+
+  // REQUIRED
+  // Timestamp of update in RFC3339, identical to create timestamp in systems
+  // that do not support updates.
+  google.protobuf.Timestamp updated = 4;
+
+  //R EQUIRED
+  // A string representing a version, some systems may use checksum, a RFC3339
+  // timestamp, or incrementing version number. For systems that do not support
+  // versioning please use your update timestamp as your version.
+  string version = 5;
+
+  //REQUIRED, the bundle SHA256 checksum which is a SHA256 checksum of all the
+  // SHA256 checksums of the files that make up this bundle
+  string checksum =  6;
+
+  //OPTIONAL
+  string dataset_id = 7;
```

Comment:
Can we document this?  Unclear why/when I would use DataBundleObject.dataset_id  vs DataBundleObject.id

Reply:
I think we should remove mentions of dataset id, it confuses the issue and adds a new concept which is out of scope for this API. This would also imply removing the search by `dataset_id` from the ListDataBundleObjects request.